CommunityShed/Pages/AddItem.aspx.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using System.Data.SqlClient;
+using System.Collections.Generic;
+using System.Linq;
+using System.Web;
+using System.Web.UI;
+using System.Web.UI.WebControls;
+using CommunityShed.Models;
+
+namespace CommunityShed
+{
+    public partial class AddItem : System.Web.UI.Page
+    {
+        protected void Page_Load(object sender, EventArgs e)
+        {
+
+        }
+
+        protected void AddItem_Click(object sender, EventArgs e)
+        {
+            string itemName = ItemNameInput.Text;
+            string warning = WarningInput.Text;
+            int itemUse = int.Parse(CategoryList.SelectedValue);
```

Comment:
I'm having difficulties mapping the variable name `itemUse` to the `CategoryList` drop down list name. Do these represent the same thing? Then, further down a bit in the SQL query, the `itemUse` variable is being used to the `@ToolId` SQL parameter. Maybe rename all of these to be consistent?